File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/iget-persons-vetting-hit_result.ts



import { GetPersonsVettingHit_Item } from './get-persons-vetting-hit_item';

export interface IGetPersonsVettingHit_Result {
  
	PersonsVettingID: number;
	VettingTypeID: number;
	HitResultID: number;
	HitResultDetails: string;
	items?: GetPersonsVettingHit_Item[];

}

File: GTTS/INL.VettingService.Models/IGetPersonsVettingHit_Result.cs
using System;
using System.Collections.Generic;
using System.Text;
using INL.Services.Models;

namespace INL.VettingService.Models
{
    public interface IGetPersonsVettingHit_Result : IBaseResult
    {
        long PersonsVettingID { get; set; }
        int VettingTypeID { get; set; }
        int HitResultID { get; set; }
        string HitResultDetails { get; set; }
        List<GetPersonsVettingHit_Item> items { get; set; }
    }
}
